drivers/net/usb/smsc95xx.h

```diff
@@ -21,7 +21,7 @@
 #define _SMSC95XX_H
 
 /* Tx command words */
-#define TX_CMD_A_DATA_OFFSET_	(0x001F0000)	/* Data Start Offset */
+#define TX_CMD_A_DATA_OFFSET_	(0x00030000)	/* Data Start Offset */
```

Comment:
What is the reason behind this change, and does it have any effect?

Reply:
It has no effect on code output, but the bitfield specifier is wrong. Edited to prevent (reduce?) confusion.